src/VerifyU2FController.js

```diff
@@ -69,25 +79,62 @@ function (Okta, FormController, FormType, FooterSignout, Q) {
       className: 'verify-u2f-form',
       noCancelButton: true,
       save: _.partial(Okta.loc, 'verify.u2f.retry', 'login'),
+      noButtonBar: function () {
+        return !window.hasOwnProperty('u2f');
+      },
       modelEvents: {
         'error': '_showRetry',
         'request': '_hideRetry'
       },
 
-      formChildren: [
-        FormType.View({
-          View: '\
+      formChildren: function () {
```

Comment:
I've noticed this before on these hello/u2f commits, but these files are almost all identical (and every PR I see shows more of the same logic being copy/pasted). 

When are we planning on doing this the right way, and moving most of this logic to shared views?